3_calibrate/src/calibration_utils.R

```diff
@@ -30,6 +30,10 @@ run_glm_cal <- function(nml_obj,
 
   # prepare to write inputs and results locally for quick I/O
   if(calibrate){ sim_dir <- file.path(sim_dir, model_group)}
+
+  # check that sim dir exists
+  if(!dir.exists(sim_dir)) dir.create(sim_dir)
+
   sim_lake_dir <- file.path(sim_dir, sprintf('%s_%s_%s', site_id, driver, time_period))
   dir.create(sim_lake_dir, recursive = TRUE, showWarnings = FALSE)
```

Comment:
RC: To set this up for future use cases where the meteo data or its extent might change, I'd delete the `sim_lake_dir` before each run if it already exists - see my PR w/ an explanation of the bug I identified and the code change [here](https://github.com/USGS-R/lake-temperature-process-models/pull/77).

Reply:
This is a reasonable suggestion, but I am only going to partially implement this.  The main reason for this is that your pipeline deletes model subfolders after successfully running and extracting results - mine does not. 

Also, I am working with a small enough subset of models that I am not worried about missing a failing model run - (1) because a failed model run will stop the pipeline, and (2) I am tracking changes to the NLDAS csvs in a [file target](https://github.com/USGS-R/lake-temperature-missouri-models/blob/907249d2cfdc17b1eb616547dc0ffba7a505b9f9/1_prep.R#L59-L63). I do have a screenshot to verify that these track as expected, but github is not letting me upload images right now!

All that being said, I _did_ go in and make `overwrite = TRUE` for all calls of `file.copy()`

Reply:
Sounds good - `overwrite=TRUE` is the key bit anyhow!